src/react/forms.jsx

```diff
@@ -5,9 +5,28 @@
 
 'use strict';
 
+var formControlClass = 'mui-form-control',
+    formGroupClass = 'mui-form-group',
+    floatingLabelBaseClass = 'mui-form-floating-label',
+    floatingLabelActiveClass = floatingLabelBaseClass + '-active',
+    animationName = 'mui-form-floating-label-inserted',
+    _supportsPointerEvents;
 
-var jqLite = require('../js/lib/jqLite.js'),
-    forms = require('../js/forms.js');
+var React = require('react');
+var cx = require('classnames');
+
+/**
+ * Check if client supports pointer events.
+ */
+function supportsPointerEvents() {
+  // check cache
+  if (_supportsPointerEvents !== undefined) return _supportsPointerEvents;
+  
+  var element = document.createElement('x');
+  element.style.cssText = 'pointer-events:auto';
```

Comment:
You can use the jqLite.css() method to get/set css properties:
https://github.com/amorey/mui/blob/master/src/js/lib/jqLite.js#L32

Here's an example:
https://github.com/amorey/mui/blob/master/src/js/forms.js#L49